IFC4x3/Properties/e/EncapsulationTechnologyCode_17B_9r6_5FWwAMTe_Dht3b/Documentation.md

```diff
@@ -0,0 +1 @@
+Code indicating the encapsulation technology which has been applied in an electric, electronic or electromechanical component.
```

Comment:
Is there a reference that can be added here or is it up to the user?

Reply:
Here up to the user! ^^